Examples/Example/Sources/Core/Custom Views/CustomCoachMarkBodyView.swift

```diff
@@ -50,7 +50,10 @@ internal class CustomCoachMarkBodyView: UIView, CoachMarkBodyView {
         self.hintLabel.font = UIFont.systemFont(ofSize: 15.0)
         self.hintLabel.isScrollEnabled = false
         self.hintLabel.textAlignment = .justified
+        #if targetEnvironment(macCatalyst)
+        #else
         self.hintLabel.layoutManager.hyphenationFactor = 1.0
```

Comment:
:+1: I'll do a sweep after this PR to remove them, I didn't realised it was deprecated in iOS 14 and unavailable in Catalyst.

Reply:
I got it!